# 17 EIA-day reaction profile

Do the four pre-2020 surprise bins rank the Wednesday RWTC return the same way after 2019?

Holdout starts 2020-01-01. Verdict: held.

The same weekly WCESTUS1 surprise as the inventory study: change minus the prior 52-week median. Train quartile edges are frozen. Each week is placed in one of four bins. The score is the Spearman correlation of the train bin-mean release return with the holdout bin-mean.

In [1]:
import json
from pathlib import Path
studies = json.loads(Path('data/studies.json').read_text())
study = next(item for item in studies['studies'] if item['slug'] == 'eia-profile')
print(study['verdict'])
print(study['verdict_line'])
print(study['table']['caption'])
for row in study['table']['rows']:
    print(' | '.join(row))


held
Train quartile edges, thousand barrels: -3060.0, 39.5, 2773.0. Spearman correlation of the bin means is 0.400. The correlation is above zero, which was the bar.
Mean Wednesday RWTC log return by inventory-surprise quartile
1 | 0.424% | 0.557%
2 | 0.301% | -0.127%
3 | -0.106% | -0.877%
4 | -0.232% | 0.240%


## Method

Quartile 1 is the most negative surprise (a larger draw, or a smaller build, than the prior 52-week median).

## Result

Train quartile edges, thousand barrels: -3060.0, 39.5, 2773.0. Spearman correlation of the bin means is 0.400. The correlation is above zero, which was the bar.